pil-analyzer/tests/condenser.rs

```diff
@@ -1,9 +1,24 @@
+use itertools::Itertools;
+use powdr_ast::analyzed::Analyzed;
 use powdr_number::GoldilocksField;
-use powdr_pil_analyzer::analyze_string;
 use test_log::test;
 
 use pretty_assertions::assert_eq;
 
+fn analyze_string(input: &str) -> Analyzed<GoldilocksField> {
+    powdr_pil_analyzer::analyze_string(input)
+        .map_err(|errors| {
```

Comment:
This code is duplicated in the other test files, but it's very difficult to share code across test binaries, so I thought it's fine.